Quick-Start-Guide/Quickstart.md

```diff
@@ -2,11 +2,11 @@ Installing GlusterFS - a Quick Start Guide
 -------
 
 #### Purpose of this document
+
 This document is intended to give you a step by step guide to setting up
 GlusterFS for the first time. For this tutorial, we will assume you are
 using Fedora 26 (or later) virtual machines.
-We also do not explain the steps in detail here as this guide is just to help
-you get it up and running as soon as possible.
+
```

Comment:
This document is intended to provide a step-by-step guide to setting up GlusterFS for the first time. For the purposes of this guide, it is required to use Fedora 26 (or, higher) virtual machine instances.